validphys2/setup.py

```diff
@@ -33,6 +33,7 @@
                         'vp-nextfitruncard = validphys.scripts.vp_nextfitruncard:main',
                         'vp-hyperoptplot = validphys.scripts.vp_hyperoptplot:main',
                         'vp-deltachi2 = validphys.scripts.vp_deltachi2:main',
+                        'get_hepdata = validphys.scripts.get_hepdata:main',
```

Comment:
I am not sure I love this name. Certainly not the underscore.

Reply:
Ah, right! For sure people can propose a better name, but it is correct that at least in order be consistent with the previous ones the underscore should be `-`.

Reply:
`vp-data` or `vp-hepdata` seem quite natural

Reply:
This sounds like a reasonable suggestion. But perhaps for more clarity we should go for `vp-getdata` (see [1ded03f](https://github.com/NNPDF/nnpdf/pull/1717/commits/1ded03f9bc616b3199765b84644e94e738b34971))?